omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -55,12 +75,15 @@ tmpdir="$(mktemp -d)/$hostname/$timestamp"
 #RHEL5 has ip and other tools here
 PATH=$PATH:/bin:/sbin
 
+PATH=/opt/$path/embedded/bin:$PATH
+
 mkdir -p "$tmpdir"
 
-for i in /opt/$path*/version-manifest.txt \
+for i in /opt/{chef,$path}*/version-manifest.txt \
     /opt/$path/pc-version.txt \
     /etc/$path/$config_name \
-    /etc/$path/chef-server-running.json \
+    /etc/${path}*/*-running.json \
+    /var/opt/$path/rabbitmq/log/rabbit@localhost.log \
```

Comment:
Good catch! There was and I think you are correct about the effect on the log file.

I'd still be :+1: on merging this since even in that case it just means we collect a little less data  and the user will see the tail failure.  

I think the right place to feed that kind of information into this script is via:

https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/templates/default/private-chef-sh.erb

which we source at the outset.